db_schema/schema.md

```diff
@@ -38,3 +38,13 @@ A record that indicates that a student is assigned to some sport activity.
 - **ActivityID** ```Integer, Foreign Key``` [(SportActivity.ID)](#sport-activity)
 
 ```Primary Key``` is the tuple (**StudentID**, **ActivityID**) since it is a many-to-many relationship.
+
+### Activity Schedule
+
+- **ActivityID** ```Integer, Foreign Key``` [(SportActivity.ID)](#sport-activity)
+- **Day** ```String(3)``` (day of the week in 3-letter format (e.g. "mon"))
+- **StartTime** ```Time, Not Null```
+- **FinishTime** ```Time, Not Null```
+- **Location** ```String(64), Not Null``` (where the training is conducted)
+
+```Primary Key``` is the tuple (**ActivityID**, **Day**).
```

Comment:
We should probably support several activities per day, so I'd suggest making a separate ID primary key

Reply:
Several schedule entries for the same activity in the same day?

Reply:
Yes

Reply:
Accepted